### PR TITLE
Updating my character information

### PR DESCRIPTION
**Pull request by jarjee**
Now brought up to date with the changes that I last had before the development freeze.

### CODE REVIEW

File: sheet/jarjee/JeanClaude.qml [hunk 1]
```diff
@@ -374,11 +390,6 @@ Item {
                 Bonus {
                     name: "Touch Penalty"
                     amount: -acArmourBonus.amount
-                },
-                Bonus {
-                    source: weararmour
```
Comment (shocklateboy92) [inline sheet/jarjee/JeanClaude.qml:379]: camelCase ids please. Also, where is wearArmour defined?

Comment (jarjee) [inline sheet/jarjee/JeanClaude.qml:379]: With all my other bonus sources all the way at the bottom of the character sheet. I'll make all the names camelCase.

File: src/attribute-manager.cpp [hunk 2]
```diff
@@ -19,6 +19,7 @@ AttributeManager &AttributeManager::instance()
 
 AttributeManager::AttributeList &AttributeManager::attributes()
 {
+    qSort(m_attributes.begin(), m_attributes.end(), [](const Resource* v1, const Resource* v2){return v1->name() < v2->name();});
```
Comment (shocklateboy92) [inline src/attribute-manager.cpp]: I don't think it makes sense to do this in the getter.
This means it'll be done every time someone wants a reference to the list.

**Comment by jarjee** [comment]
The alternative is when someone inserts into the list, which means a lot of redundant sorting unless QT has a sorted list primitive that does insertion sorts.

**Comment by shocklateboy92** [comment]
Well, AttributeManager is going to be deprecated soon anyway.
I think we can find a way to sort it in between those steps.

Either way, ditch this change. We'll merge in the rest of your character sheet changes, and figure out how to best address this later.

**Comment by jarjee** [comment]
And fixed. Any day that requires reflog is a good day.

**Comment by shocklateboy92** [comment]
You mean any day that _doesn't_ require a reflog is a good day ;-)

**Pull request merged**